src/app/draft/components/selected-markets.tsx

```
@@ -0,0 +1,30 @@
+import { Link, List, ListItem } from '@chakra-ui/react'
+
+export type SelectedMarkets = {
+  title: string
+  id: number
+}
+export type SelectedMarketsProps = {
+  market: SelectedMarkets[]
+}
+
+export const SelectedMarkets = ({ market }: SelectedMarketsProps) => {
+  return !!market.length ? (
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)